src/main/java/io/snyk/jenkins/SnykStepBuilder.java

```diff
@@ -277,60 +270,55 @@ public boolean perform(@Nonnull AbstractBuild<?, ?> build, @Nonnull Launcher lau
 
       String monitorUri = "";
       if (monitorProjectOnBuild) {
-        FilePath snykMonitorReport = workspace.child(SNYK_MONITOR_REPORT_JSON);
-        FilePath snykMonitorDebug = workspace.child(SNYK_MONITOR_REPORT_JSON + ".debug");
-
-
         ArgumentListBuilder argsForMonitorCommand = buildArgumentList(snykExecutable, "monitor", envVars);
+        ByteArrayOutputStream monitorStdout = new ByteArrayOutputStream();
 
-        log.getLogger().println("Remember project for continuous monitoring...");
-        log.getLogger().println("> " + argsForMonitorCommand);
-        int monitorExitCode;
-        try (
-          OutputStream snykMonitorOutput = snykMonitorReport.write();
-          OutputStream snykMonitorDebugOutput = snykMonitorDebug.write()
-        ) {
-          monitorExitCode = launcher.launch()
-            .cmds(argsForMonitorCommand)
-            .envs(envVars)
-            .stdout(snykMonitorOutput)
-            .stderr(snykMonitorDebugOutput)
-            .quiet(true)
-            .pwd(workspace)
-            .join();
-        }
-        String snykMonitorReportAsString = snykMonitorReport.readToString();
+        logger.println("Remember project for continuous monitoring...");
+        logger.println("> " + argsForMonitorCommand);
+        int monitorExitCode = launcher.launch()
+          .cmds(argsForMonitorCommand)
+          .envs(envVars)
+          .stdout(monitorStdout)
+          .stderr(logger)
+          .quiet(true)
+          .pwd(workspace)
+          .join();
+        String snykMonitorReportAsString = monitorStdout.toString(UTF_8.name());
         if (monitorExitCode != 0) {
-          log.getLogger().println("Warning: 'snyk monitor' was not successful. Exit code: " + monitorExitCode);
-          log.getLogger().println(snykMonitorReportAsString);
+          logger.println("Warning: 'snyk monitor' was not successful. Exit code: " + monitorExitCode);
+          logger.println(snykMonitorReportAsString);
         }
 
         if (LOG.isTraceEnabled()) {
           LOG.trace("Command line arguments: {}", argsForMonitorCommand);
           LOG.trace("Exit code: {}", monitorExitCode);
           LOG.trace("Command standard output: {}", snykMonitorReportAsString);
-          LOG.trace("Command debug output: {}", snykMonitorDebug.readToString());
         }
 
         SnykMonitorResult snykMonitorResult = ObjectMapperHelper.unmarshallMonitorResult(snykMonitorReportAsString);
         if (snykMonitorResult != null && fixEmptyAndTrim(snykMonitorResult.uri) != null) {
-          log.getLogger().println("Explore the snapshot at " + snykMonitorResult.uri);
+          logger.println("Explore the snapshot at " + snykMonitorResult.uri);
```

Comment:
As `monitorUri` isn't really used, we can just pass `snyk monitor`'s default log output to Jenkins' console output instead of dealing with JSON and logging a similar output.